doc/admin/deployment.md

```diff
@@ -825,7 +825,8 @@ support is to pass the -i flag to daos_agent.
 The `daos_agent` configuration file is parsed when starting the
 `daos_agent` process. The configuration file location can be specified
 on the command line (`daos_agent -h` for usage) or default location
-(`install/etc/daos_agent.yml`).
+(`install/etc/daos_agent.yml`). If installed from rpms the default location is
+(`/usr/etc/daos_agent.yml`).
```

Comment:
Is this correct? When I used the RPMs for my demo the agent was reading configs from /etc/daos/ as the default location.

Reply:
Yes the location did appear to change.  I can make the edit